gtk4/sys/src/lib.rs

```diff
@@ -8055,6 +8112,7 @@ extern "C" {
     // GtkOrdering
     //=========================================================================
     pub fn gtk_ordering_get_type() -> GType;
+    pub fn gtk_ordering_from_cmpfunc(cmpfunc_result: c_int) -> GtkOrdering;
```

Comment:
Missing `Since`

Reply:
This one was always there, just wasn't inside `#ifdef __GI_SCANNER__`. I think we don't need it here, we already have From/Into GtkOrdering to Rust's ordering enum. Maybe could be used inside a From<i32> implementation.